src/test/java/nextstep/ladder/LineTest.java

```diff
@@ -0,0 +1,25 @@
+package nextstep.ladder;
+
+import nextstep.ladder.domain.Line;
+import nextstep.ladder.strategy.RandomStrategy;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+@Nested
+@DisplayName("Line 클래스는")
```

Comment:
```suggestion
import java.util.ArrayList;
import java.util.List;

@Nested
@DisplayName("Line 클래스는")
```

`@Nested` 시도는 좋지만 현재는 테스트 메서드가 하나네요. 🤔

Reply:
앞으로 테스트 코드가 증가할 것 같아서 일단 이렇게 짰는데 수정할까요?!

Reply:
추가된다면 그 시점부터 시작하는 것은 어떨까요?